DolbyIO/Source/DolbyIOModule/Public/DolbyIO.h

```diff
@@ -28,62 +28,89 @@ class DOLBYIOMODULE_API UDolbyIO : public UGameInstanceSubsystem, public DolbyIO
 	 *
 	 */
 
-	/** Initializes or refreshes the client access token. Initializes the plugin unless already initialized. Triggers
-	 * OnInitialized if initialization was successful.
-	 * @param Token - Client access token.
+	/** Initializes or refreshes the client access token. The function takes the token as a parameter and initializes the plugin unless already initialized. Successful initialization triggers the [On Initialized](#on-initialized) event.
+	 *
+	 * For quick testing, you can manually obtain a token from the [Dolby.io dashboard](https://dashboard.dolby.io/) and paste it directly into the node.
+	 *
+	 * You may use the [Set Token Using Key and Secret](#set-token-using-key-and-secret) function instead for convenience during onboarding.
+	 *
+	 * Example:
+	 * ![example](./../../../../Images/on_token_needed.PNG)
```

Comment:
These examples probably won't be visible in Unreal Editor. I do think there's value in trying to make it possible to automatically generate documentation based on this file, so that it becomes the ultimate source of information on the plugin and therefore putting the example image links inside it makes some sense, but I'm not sure about it and definitely not about using relative paths like this. Using relative paths with look silly to someone just looking at the code and will simply not work when we put the plugin on Epic Marketplace. The plugin's code will be there, but the images will not, so they would at least have to be linked to specific files on the Internet.